app/controllers/contact_topics_controller.rb

```diff
@@ -0,0 +1,66 @@
+class ContactTopicsController < ApplicationController
+  before_action :set_contact_topic, only: %i[edit update destroy]
+  after_action :verify_authorized
+
+  # GET /contact_topics/new
+  def new
+    authorize ContactTopic
+    contact_topic = ContactTopic.new(casa_org_id: current_user.casa_org_id)
+    contact_topic.contact_topic_detail = ContactTopicDetail.new
+    contact_topic.contact_topic_question = ContactTopicQuestion.new
+    @contact_topic = contact_topic
+  end
+
+  # GET /contact_topics/1/edit
+  def edit
+    authorize @contact_topic
+  end
+
+  # POST /contact_topics or /contact_topics.json
+  def create
+    authorize ContactTopic
+    @contact_topic = ContactTopic.new(contact_topic_params)
+
+    if @contact_topic.save
+      redirect_to edit_casa_org_path(current_organization), notice: "Contact topic was successfully created."
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /contact_topics/1 or /contact_topics/1.json
+  def update
+    authorize @contact_topic
+
+    if @contact_topic.update(contact_topic_params)
+      redirect_to edit_casa_org_path(current_organization), notice: "Contact topic was successfully updated."
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /contact_topics/1 or /contact_topics/1.json
+  def destroy
+    authorize @contact_topic
+
+    if @contact_topic.update(active: false)
+      redirect_to edit_casa_org_path(current_organization), notice: "Contact topic was successfully removed."
```

Comment:
After talking to Malory and Brandon, I added a soft delete boolean and added being able to change active to the edit view. This controller action now soft deletes, I changed the action. But I have kept the language of the alert the same because I think its an implementation details that it soft deletes vs actually removing a record.